<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SCore Detector</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>-4</y>
      <width>831</width>
      <height>411</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>241</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>831</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>490</y>
      <width>571</width>
      <height>91</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Comments..</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>590</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Click here to add comment</string>
    </property>
    <property name="text">
     <string>Submit Comment</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_object">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>500</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>520</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Objects =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>time =</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>590</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Train new Classifier</string>
    </property>
    <property name="text">
     <string>Train classifier</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>420</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QTextEdit" name="edit_port">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Type prot number of device</string>
     </property>
     <property name="placeholderText">
      <string>Enter port number ....</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLoadStream">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Click here to load video</string>
     </property>
     <property name="text">
      <string>Load Stream</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>IP Address</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditAdd">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>70</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Type IP address of device</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Enter IP address....</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>30</y>
      <width>141</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Get ful report about video</string>
     </property>
     <property name="text">
      <string>View Report</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="_chk_har">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Select algorithm as Haar</string>
     </property>
     <property name="text">
      <string>Haar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPlay">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Play/Pause</string>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLoadVideo">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Load Video</string>
     </property>
     <property name="text">
      <string>Load Video</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btncascade">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Browse Classifiers</string>
     </property>
     <property name="text">
      <string>Cascade</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>96</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>select Classifier</string>
     </property>
     <item>
      <property name="text">
       <string>haarcascadeeye.xml</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>haarcascadefrontalface.xml</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Select algorithm as Hog</string>
     </property>
     <property name="text">
      <string>Hog</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1126</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
